src/distributed_scheduler.rs

```diff
@@ -129,19 +129,14 @@ impl DistributedScheduler {
         //TODO accumvalues needs to be done
     ) {
         let result = Some(result);
-        match event_queues.lock().get_mut(&(task.get_run_id())) {
-            Some(queue) => {
-                queue.push_back(CompletionEvent {
-                    task,
-                    reason,
-                    //                    result: Some(Box::new(result)),
-                    result: result,
-                    accum_updates: HashMap::new(),
-                });
-            }
-            None => {
-                //                println!("ignoring completion event for DAG Job");
```

Comment:
Same as the local scheduler, in None case, we should log whatever is inside println statement. Use info! macro instead of println.